config/webhook/manifests.yaml

```
@@ -159,6 +159,27 @@ webhooks:
     resources:
     - clusterclasses
   sideEffects: None
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
+      path: /delete-hook-cluster-x-k8s-io-v1beta1-cluster
+  failurePolicy: Fail
+  matchPolicy: Equivalent
+  name: delete-hook.cluster.x-k8s.io
```

```suggestion
  name: delete-hook.cluster.cluster.x-k8s.io
```
nit: to follow this pattern: default.cluster.cluster.x-k8s.io